### PR TITLE
some fixes for streamers

### PR DESCRIPTION
**Pull request by fibnumb**

### CODE REVIEW

**Comment by fibnumb** [comment]
Could someone please start the test?

Thanks,
AC

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx [hunk 1]
```diff
@@ -67,21 +67,21 @@ using std::endl;
  */
 AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA::AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA() :
 AliAnalysisTaskEmcalJet(),
-fUseRecalcPatches(false),
+fUseRecalcPatches(0x0),
```
Comment (mfasDa) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx]: This is a change with no effect: The compiler will perform an implicit type conversion and replace the hex value with true. I recommend not to use numeric values like hex values for booleans as this gets confusing.

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.h [hunk 2]
```diff
@@ -107,32 +107,22 @@ class AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA : public AliAnalysisTaskEmcalJ
     //void                        DoMCJetLoop()                                     ;
     //Bool_t                      IsLEDEvent() const                                ;
     
-    Bool_t                      fUseRecalcPatches                                 ;///<                  Switch between offline (FEE) and recalc (L1) patches
+    Bool_t                      fUseRecalcPatches                                 ;//                  Switch between offline (FEE) and recalc (L1) patches
```
Comment (mfasDa) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.h]: This is as well a change without effect: This doxygen comment is transparent to both ROOT5 and ROOT6 and will therefor be ignored by rootcint/rootcling.

**Comment by mfasDa** [comment]
+test

**Comment by fibnumb** [comment]
I changed these back

**Comment by mfasDa** [comment]
+1

**Pull request merged**